src/doc_builder/commands/convert_doc_file.py

```diff
@@ -68,8 +68,14 @@ def convert_command(args):
     with open(source_file, "r", encoding="utf-8") as f:
         text = f.read()
 
-    page_info = {"package_name": package_name, "page": source_file.with_suffix(".html").relative_to(doc_folder)}
+    page_info = {
+        "package_name": package_name,
+        "page": source_file.with_suffix(".html").relative_to(doc_folder),
+        "no_prefix": True,
+    }
     text = convert_rst_to_mdx(text, page_info, add_imports=False)
+    text = text.replace("&amp;lcub;", "{")
+    text = text.replace("&amp;lt;", "<")
```

Comment:
When converting back those Markdown files, this will be undoe.

Reply:
could you elaborate on this?

Reply:
The source files in the doc should not have those special chars as we don't expect the users to write the doc this way (we're inside the command that converts an rst to the new expected doc format). We apply the conversion for those files in the `doc_builder` in [this function](https://github.com/huggingface/doc-builder/blob/66c63fdc291717fc40f0c2b19aa87390f6a5aafa/src/doc_builder/convert_md_to_mdx.py#L39).

Reply:
oh I see now